lib/rules/require-ember-lifeline.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @fileOverview Require the use of ember-lifeline's runTask over using Ember.run.later.
+ * @author Steve Calvert
+ */
+
+const { getCaller, cleanCaller } = require('../utils/caller');
+const { getEmberImportBinding } = require('../utils/imports');
+const { collectObjectPatternBindings } = require('../utils/destructed-binding');
+
+let LATER_PATTERNS = ['Ember.run.later', 'run.later'];
+
+const MESSAGE = 'Please use runTask from ember-lifeline in favor of Ember.run.later.';
+
+module.exports = {
+  docs: {
+    description: 'Please use runTask from ember-lifeline in favor of Ember.run.later.',
+    category: 'Best Practices',
+    recommended: false
+  },
+  meta: {
+    message: MESSAGE
+  },
+  create(context) {
+    let emberImportBinding;
+
+    return {
+      ObjectPattern(node) {
+        if (emberImportBinding) {
+          LATER_PATTERNS = LATER_PATTERNS.concat(collectObjectPatternBindings(node, {
+            [emberImportBinding]: ['run']
+          }));
+        }
+      },
+
+      ImportDefaultSpecifier(node) {
```

Comment:
Nit: Can we move this above `ObjectPattern`? Since it is expected to run prior to that function.